src/__internal__/input/input.component.tsx

```diff
@@ -17,8 +17,13 @@ export interface CommonInputProps
   extends Omit<React.InputHTMLAttributes<HTMLInputElement>, "type"> {
   /* The default value alignment on the input */
   align?: "right" | "left";
-  /** The id of the element that describe the input. */
+  /**
```

Comment:
comment: I think it's worth following the same pattern as we do with other deprecations and using `Logger.deprecate` here with a flag to prevent it called twice and a test etc

Reply:
It might need to be in Textbox and Textarea if we go with @Parsium's suggestion about passing it into the combined array thinking about it